src/Core/Content/Product/SalesChannel/Detail/ProductDetailRoute.php

```diff
@@ -107,11 +110,12 @@ public function load(string $productId, Request $request, SalesChannelContext $c
                 $this->breadcrumbBuilder->getProductSeoCategory($product, $context)
             );
 
-            $configurator = $this->configuratorLoader->load($product, $context);
+            $skipConfigurator = $request->query->getBoolean(self::SKIP_CONFIGURATOR);
+            $configurator = !$skipConfigurator ? $this->configuratorLoader->load($product, $context) : null;
```

Comment:
personal opinion for both variables: invert the boolean value while defining the variables and rename them accordingly. imo better readable on the place, where they are used. 


```suggestion
            $loadConfigurator = !$request->query->getBoolean(self::SKIP_CONFIGURATOR);
            $configurator = $loadConfigurator ? $this->configuratorLoader->load($product, $context) : null;
```

Reply:
@mitelg I changed it like how you requested it.